src/root/components/ThreeWForm/useThreeWOptions.ts

```diff
@@ -137,7 +137,7 @@ export const schema: FormSchema = {
       primary_sector: [requiredCondition],
       programme_type: [requiredCondition],
       project_country: [requiredCondition],
-      project_districts: [requiredListCondition],
+      project_districts: [],
```

Comment:
Let's also verify that backe-end accepts empty value or empty list for this, If we want to send the empty list, we have to add defaultEmptyListCondition or something like that exists